tests/ci/docker_images/linux-aarch/push_images.sh

```diff
@@ -13,10 +13,10 @@ echo "Uploading docker images to ${ECS_REPO}."
 $(aws ecr get-login --no-include-email --region us-west-2)
 
 # Tag images with date to help find old images, CodeBuild uses the latest tag and gets updated automatically
-docker tag amazonlinux-2-aarch:gcc-7x ${ECS_REPO}:amazonlinux-2-aarch_gcc-7x_`date +%Y-%m-%d`
-docker tag amazonlinux-2-aarch:gcc-7x ${ECS_REPO}:amazonlinux-2-aarch_gcc-7x_latest
-docker push ${ECS_REPO}:amazonlinux-2-aarch_gcc-7x_latest
-docker push ${ECS_REPO}:amazonlinux-2-aarch_gcc-7x_`date +%Y-%m-%d`
```

Comment:
`amazonlinux-2-aarch_gcc-7x_latest` is the actual one used by `github_ci_linux_arm_omnibus.yml`. To fix typo in `run-cdk.sh`, `amazonlinux-2_gcc-7x_latest` can be renamed.

Reply:
Gonna fix Omnibus file then. I'm gonna take down CI and bring up new one when I add the Valgrind image.

Reply:
ok. `take down` is not necessary. Only docker image rebuilt and corresponding build spec change are needed.